### PR TITLE
N44: rust-gpu-sdf

### PR DESCRIPTION
**Pull request by Shfty**
_Part of #1317_

Adds `rust-gpu-sdf` section to newsletter.

### CODE REVIEW

**Comment by AngelOnFira** [comment]
This section is too long I think, coming in at 2000 characters on the rendered version:

image

I think that it is pretty condensed though, and it is neat to read. What are your thoughts @ozkriff? I'm thinking of leaving it as is, or just getting it to be decreased to ~1500 characters instead.

**Comment by Shfty** [comment]
It is a bit wordy -  if necessary I could probably chop it down without losing too much by merging paragraphs 1/2 and 3/4 and reducing the applications / implementation with some context links to wikipedia or such.

**Review by ozkriff — CHANGES_REQUESTED** [review]
Yeah, I think it's worth cutting the section a bit, currently it feels too long and detailed for the usual newsletter format

**Comment by Shfty** [comment]
I've edited it down to ~1200 characters - it's not quite so explicit about the applications and implementation details, but I've added a couple of links that should make up the difference for readers who aren't familiar with the field.

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Tnx!
